src/EditorFeatures/CSharpTest/Organizing/OrganizeTypeDeclarationTests.cs

```diff
@@ -20,276 +20,300 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Organizing
     public class OrganizeTypeDeclarationTests : AbstractOrganizerTests
     {
         [Fact, Trait(Traits.Feature, Traits.Features.Organizing)]
-        public async Task TestFieldsWithoutInitializers1()
+        [InlineData("class")]
+        [InlineData("record")]
+        public async Task TestFieldsWithoutInitializers1(string typeKind)
         {
             var initial =
-@"class C {
+$@"{typeKind} C {{
     int A;
     int B;
     int C;
-}";
+}}";
 
             var final =
-@"class C {
+$@"{typeKind} C {{
     int A;
     int B;
     int C;
-}";
+}}";
             await CheckAsync(initial, final);
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.Organizing)]
-        public async Task TestFieldsWithoutInitializers2()
+        [InlineData("class")]
+        [InlineData("record")]
+        public async Task TestFieldsWithoutInitializers2(string typeKind)
         {
             var initial =
-@"class C {
+$@"{typeKind} C {{
     int C;
     int B;
     int A;
-}";
+}}";
 
             var final =
-@"class C {
+$@"{typeKind} C {{
     int A;
     int B;
     int C;
-}";
+}}";
```

Comment:
@CyrusNajmabadi This looks like a nice feature that I was never aware it's existed. Can I ask where this feature is in Visual Studio? I can't find any option along the words "organize declarations"